src/STRPAK.CPP

```diff
@@ -11,6 +11,16 @@
 
 #include <messages.h>	// MAX_MSGLEN msgI
 
+#if __has_include(<filesystem>)
+#include <filesystem>
+namespace filesys = std::filesystem;
+#elif __has_include(<experimental/filesystem>)
+#include <experimental/filesystem>
+namespace filesys = std::experimental::filesystem;
+#else
+#error "no filesystem support"
+#endif
```

Comment:
We already made a decision to keep filesystem isolated to a single compilation unit. If you need new capability from filesystem, wrap it in a function in the existing compilation unit.

Reply:
Done.